setup.py

```diff
@@ -74,6 +74,7 @@ def local_file(name):
     zip_safe=False,
     extras_require=extras,
     install_requires=install_requires,
+    python_requires='==2.7,>=3.4',
```

Comment:
Whoops, fixed.  I guess this is an example of why we should be very careful with #454.